src/components/examples/footer.vue

```diff
@@ -25,13 +25,19 @@ export default {
 </style>
 
 <style lang="stylus">
++layout(xs)
+  .ex-attributes .zi-card
+    display flex
```

Comment:
There may be side effects for multiple table examples.

![demo (1)](https://user-images.githubusercontent.com/11304944/76386119-912f4c00-639e-11ea-92bb-eb25f766f65a.png)